etl/steps/data/garden/wb/2024-06-26/poverty_projections.meta.yml

```diff
@@ -0,0 +1,101 @@
+# NOTE: To learn more about the fields, hover over their names.
+definitions:
+  common:
+    processing_level: minor
+    presentation:
+      topic_tags:
+        - Poverty
+  description_key_estimated_extreme_poverty: Extreme poverty here is defined as living below the International Poverty Line of $2.15 per day.
```

Comment:
Is there a benefit to making all the key points separate? The first four points always show up together in the descriptions below, if possible, I would combine them in one variable.

Reply:
No, there's no benefit. I thought any option would break the parameter, but I applied anchors and they worked. Thanks!